conf/application-json-logger.xml

```diff
@@ -7,8 +7,6 @@
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
 
-    <logger name="eori-common-component-frontend" level="${logger.eori-common-component-frontend:-${logger.application:-WARN}}"/>
```

Comment:
There is a setting for uk.gov logger above so this logger will cover the loggers created by me.